db/migrate/20230208020404_populate_category_settings.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class PopulateCategorySettings < ActiveRecord::Migration[7.0]
+  def up
+    execute(<<~SQL)
+      INSERT INTO
+        category_settings(
+          category_id,
+          require_topic_approval,
+          require_reply_approval,
+          num_auto_bump_daily,
+          created_at,
+          updated_at
+        )
+      SELECT
+        category_id,
+        MAX(
```

Comment:
It might seem weird that we're doing `MAX` here, but by right there should only be one setting per key, so we use this as a make-shift way of getting that one value. It works since `MAX` operates on many column types.